cmstestsuite/unit_tests/grading/__init__.py

```diff
@@ -22,10 +22,16 @@
 """
 
 from __future__ import absolute_import
+from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+from future.builtins.disabled import *
+from future.builtins import *
 
-import StringIO
+try:
+    from BytesIO import BytesIO
+except ImportError:
+    from io import BytesIO
```

Comment:
No need: `io.BytesIO` exists in both py2 and py3.

Reply:
Done.